test/unittest/utils/CMakeLists.txt

```diff
@@ -111,7 +111,11 @@ target_include_directories(LocatorTests PRIVATE
     ${PROJECT_SOURCE_DIR}/include ${PROJECT_BINARY_DIR}/include ${Asio_INCLUDE_DIR}
     ${PROJECT_SOURCE_DIR}/src/cpp)
 target_link_libraries(LocatorTests GTest::gtest ${MOCKS})
-add_gtest(LocatorTests SOURCES ${LOCATORTESTS_SOURCE})
+if(EPROSIMA_TEST_DNS_NOT_SET_UP)
```

Comment:
Not sure this was intented for this PR. Please check.

Reply:
see above.